exec.go

```diff
@@ -119,6 +119,14 @@ func execProcess(context *cli.Context) (int, error) {
 	if err != nil {
 		return -1, err
 	}
+
+	// convert pid-file to an absolute path so we can write to the right
+	// file after chdir to bundle
+	pidFile, err := getPidFile(context)
+	if err != nil {
+		return -1, err
+	}
```

Comment:
@cyphar Thanks, I think that all of the three command can use `ctx.Set` method to revise `pid-fle`

I have updated the code and make some code refactoring.